onnxruntime/core/graph/graph.cc

```diff
@@ -4062,7 +4062,11 @@ void Graph::ToGraphProtoInternal(ONNX_NAMESPACE::GraphProto& graph_proto) const
     *(graph_proto.mutable_output()->Add()) = output_arg->ToProto();
   }
 
-  for (const auto* value_info : value_info_) {
+  auto sort_predicate = [](const NodeArg* v1, const NodeArg* v2) {return v1->Name() < v2->Name();};
```

Comment:
Linter reports an error here. Can run `lintrunner -a` in the root `onnxruntime/` directory to fix these.
```c++
  auto sort_predicate = [](const NodeArg* v1, const NodeArg* v2) {  return v1->Name() < v2->Name(); };
```

Reply:
Fixed.